show/kube.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/python -u
+# -*- coding: utf-8 -*-
+    
+import os
+
+def hostname():
```

Comment:
Duplicated code should be moved into swsssdk. Or try similar function `socket.gethostname()` ?

Reply:
swsssdk is deprecated. swsscommon is the new standard.

Most of these functions don't necessarily belong in swsscommon; they belong in a shared library in sonic-utilities which all utilities can import.

Reply:
This helps override the hostname for kube use, as needed.

Reply:
Even if we fix on hostname, again it is part of re-factoring.